Bottom: 68522071f9dffbc9bcf08122c9196cc8c0998f04
Top:    b502eb5f207b7557cfd406c8851a6685b108c288
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 18:52:06 +0000

Refresh of win32-code-fixes-to-be-split

---

diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index 6879a98..f802839 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -55,6 +55,7 @@ start_ap214_file (const char *filename)
   FILE *f;
   time_t currenttime;
   struct tm utc;
+  struct tm *tmp;
 
   f = fopen (filename, "w");
   if (f == NULL)
@@ -64,7 +65,9 @@ start_ap214_file (const char *filename)
     }
 
   currenttime = time (NULL);
-  gmtime_r (&currenttime, &utc);
+//  gmtime_r (&currenttime, &utc);
+  tmp =  gmtime (&currenttime);
+  utc = *tmp;
 
   fprintf (f, "ISO-10303-21;\n");
   fprintf (f, "HEADER;\n");
diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index c109dc5..4b43d07 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -132,8 +132,11 @@ step_start_file (FILE *f, const char *filename)
 {
   time_t currenttime = time (NULL);
   struct tm utc;
+  struct tm *tmp;
 
-  gmtime_r (&currenttime, &utc);
+//  gmtime_r (&currenttime, &utc);
+  tmp =  gmtime (&currenttime);
+  utc = *tmp;
 
   fprintf (f, "ISO-10303-21;\n");
   fprintf (f, "HEADER;\n");
